pkg/result/processors/exclude_rules.go

```diff
@@ -118,3 +118,36 @@ func (ExcludeRules) Name() string { return "exclude-rules" }
 func (ExcludeRules) Finish()      {}
 
 var _ Processor = ExcludeRules{}
+
+type ExcludeRulesCaseSensitive struct {
+	*ExcludeRules
+}
+
+func NewExcludeRulesCaseSensitive(rules []ExcludeRule, lineCache *fsutils.LineCache, log logutils.Log) *ExcludeRulesCaseSensitive {
```

Comment:
I've expected a `caseSensitive bool` parameter here to aviod code duplication.
Boolean params is anti-pattern, but I think that we can make an exception here.

Reply:
I think we do not need boolean parameter if we have separate constructors.